### PR TITLE
project1 registration

### PR DESCRIPTION
**Pull request by dharshakch97-cloud**
Yesterday's task registration part in project 1 was completed. See this request and review my code.

### CODE REVIEW

File: project1/application.py [hunk 1]
```diff
@@ -46,23 +49,26 @@ def show_result():
         user_email=request.form['email']
         user_password=request.form['psw']
         new_user = Users(username=user_name, email=user_email, password=user_password)
```
Comment (rehana-ms) [inline project1/application.py:51]: timestamp field is there in model.But I see you are not adding it while inserting your record.

File: project1/import.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+def main():
+    db.create_all()
+    with open("books.csv", 'r') as f:
+        reader = csv.reader(f)
+        next(reader)
+
+        for row in reader:
+            newbook = Book(row[0],row[1],row[2],int(row[3]))
+            db.session.add(newbook)
+        db.session.commit()
```
Comment (rehana-ms) [inline project1/import.py:22]: Are you able to load all records into the table?

Comment (dharshakch97-cloud) [reply project1/import.py:22]: Yes! all of the records in the book.csv file imported to the book database.

File: project1/application.py (hunk 1 above)
Comment (dharshakch97-cloud) [reply project1/application.py:51]: The timestamp field is declared in the class, and when we initialized the timestamp added to the database.
image

File: project1/import.py (hunk 2 above)
Comment (rehana-ms) [reply project1/import.py:22]: ok good.

File: project1/application.py (hunk 1 above)
Comment (rehana-ms) [reply project1/application.py:51]: ok.once can you share database screenshot.

Comment (dharshakch97-cloud) [reply project1/application.py:51]: image

Comment (dharshakch97-cloud) [reply project1/application.py:51]: image
This one is users database.

Comment (rehana-ms) [reply project1/application.py:51]: Screenshot for the data plz.

Comment (rehana-ms) [reply project1/application.py:51]: In users table.

Comment (dharshakch97-cloud) [reply project1/application.py:51]: image

Comment (dharshakch97-cloud) [reply project1/application.py:51]: image

**Comment by rehana-ms** [comment]
good to merge.

**Comment by dharshakch97-cloud** [comment]
Reopen to merge branch

**Pull request merged**